### PR TITLE
Add content to working mode page

### PR DESCRIPTION
**Pull request by brentzundel**
Working mode: line length, link fixes, minor edits, added content from 01-10-19 meeting

The changes will be more easily reviewed using `rich diff` rather than `source diff`

Signed-off-by: Brent <brent.zundel@gmail.com>

### CODE REVIEW

File: WorkMode/guiding_principles.md [hunk 1]
```diff
@@ -17,63 +30,105 @@ This document is a *Living Document* and as such will change. Members of the gro
...
+World and Closed World, if something is not defined by the specification
+then it is permissible (just not standardized) rather than being disallowed.
 
 **Notes**
 
-* This is hard to judge when it is appropriate to be prescriptive, and when it is appropriate to be silent. Several issues have arisen on both sides -- sometimes 1.0 is too prescriptive, sometimes the definition is not precise enough.
+* It is hard to judge when it is appropriate to be prescriptive, and when
+    it is appropriate to be silent. Several issues have arisen on both
+    sides -- sometimes 1.0 is too prescriptive, sometimes the definition is
+    not precise enough.
```
Comment (burnburn) [inline WorkMode/guiding_principles.md]: Not clear which 1.0 this is referencing.  Perhaps drop the final sentence?

**Review by burnburn — CHANGES_REQUESTED** [review]
I may suggest some other changes later via separate PR to keep things simple (and get this merged), but this is fine after my one comment is addressed.

File: WorkMode/guiding_principles.md [hunk 2]
```diff
@@ -17,63 +30,105 @@ This document is a *Living Document* and as such will change. Members of the gro
...
+the easier to have non-breaking changes in the future and the easier it is
+to have experimentation. In a similar way to the distinction between Open
+World and Closed World, if something is not defined by the specification
+then it is permissible (just not standardized) rather than being disallowed.
 
 **Notes**
 
-* This is hard to judge when it is appropriate to be prescriptive, and when it is appropriate to be silent. Several issues have arisen on both sides -- sometimes 1.0 is too prescriptive, sometimes the definition is not precise enough.
+* It is hard to judge when it is appropriate to be prescriptive, and when
+    it is appropriate to be silent. Several issues have arisen on both
```
Comment (brentzundel) [inline WorkMode/guiding_principles.md]:
```suggestion
    it is appropriate to be silent.
```

File: WorkMode/guiding_principles.md [hunk 3]
```diff
@@ -17,63 +30,105 @@ This document is a *Living Document* and as such will change. Members of the gro
...
+    sides -- sometimes 1.0 is too prescriptive, sometimes the definition is
```
Comment (brentzundel) [inline WorkMode/guiding_principles.md]:
```suggestion
```

File: WorkMode/guiding_principles.md (hunk 1 above)
Comment (brentzundel) [inline WorkMode/guiding_principles.md]:
```suggestion
```

**Comment by brentzundel** [comment]
@burnburn I removed the sentence.

**Comment by brentzundel** [comment]
@burnburn I would really like to get this merged. Are there any additional changes you see are needed?

File: WorkMode/guiding_principles.md [hunk 4]
```diff
@@ -5,9 +5,22 @@ layout: default
...
+Working Group to assist in making decisions about specification features.
+These guidelines provide a framework in which to have productive
+discussions that quickly come to consensus, rather than ending up blocked
+in an unresolvable tug-of-war over opposing and equally valid points of
+view. The principles are not meant to be interpreted as strict or objective
+rules, but instead as informative of the overall direction of the group to
+promote consistency and usability of the resulting specifications. 
+
+This document is a *Living Document* and as such will change. Members of
+the group are encouraged to edit (e.g. to update, correct, etc.) the
```
Comment (TallTed) [inline WorkMode/guiding_principles.md]:
```suggestion
the group are encouraged to edit (e.g., to update, correct, etc.) the
```

File: WorkMode/guiding_principles.md [hunk 5]
```diff
@@ -17,63 +30,103 @@ This document is a *Living Document* and as such will change. Members of the gro
...
+As the mission is interoperability, having at least two supporters for each
+use case is important. If only one supporter can be found, then any
+features that it would require are likely too specific for inclusion in the
+proposed form.
 
 **Notes**
 
-* Any independent organization, group or individual can be a supporter, regardless of whether they are part of the Working Group, or even members of the W3C. Consortia or other standards organizations that represent many participants can thus fulfill this principle via their members.
-* This is orthogonal to implementations of the specifications, which are required to exit Candidate Recommendation phase.
+* Any independent organization, group or individual can be a supporter,
```
Comment (TallTed) [inline WorkMode/guiding_principles.md]:
```suggestion
* Any independent organization, group, or individual can be a supporter,
```

File: WorkMode/guiding_principles.md [hunk 6]
```diff
@@ -17,63 +30,103 @@ This document is a *Living Document* and as such will change. Members of the gro
...
+    regardless of whether they are part of the Working Group, or members of
```
Comment (TallTed) [inline WorkMode/guiding_principles.md]:
```suggestion
    regardless of whether they are part of the Working Group, or a member of
```

File: WorkMode/guiding_principles.md [hunk 7]
```diff
@@ -17,63 +30,103 @@ This document is a *Living Document* and as such will change. Members of the gro
...
+    the W3C. Consortia or other standards organizations that represent many
+    participants can thus fulfill this principle via their members.
+* This is orthogonal to implementations of the specifications, which are
+    required to exit Candidate Recommendation phase.
 
 ## As simple as possible, but no simpler.
 
-A simpler solution (to understand, implement and use) is better than a more complex solution that accomplishes the same result.
+A simpler solution (to understand, implement and use) is better than a more
```
Comment (TallTed) [inline WorkMode/guiding_principles.md]:
```suggestion
A solution that is simpler (to understand, implement, and use) is better than a more
```

File: WorkMode/index.md [hunk 8]
```diff
@@ -5,38 +5,67 @@ layout: default
...
+
+Note that the [WG's Charter](https://www.w3.org{{ site.charter }}) formally
+defines the general framework of the group's working mode. In all cases,
+the Charter and/or the [W3C Process Document](//www.w3.org/Consortium/Process/) 
+overrides the information in this document. Nevertheless, this document
+contains additional information about how the group *really* works, so this
+information may be particularly useful to new members of the group.
+
+This document is a *Living Document* and as such will change. Members of
+the group are encouraged to edit (e.g. to update, correct, etc.) the
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
the group are encouraged to edit (e.g., to update, correct, etc.) the
```

File: WorkMode/index.md [hunk 9]
```diff
@@ -5,38 +5,67 @@ layout: default
...
 * Contributing tests for the group’s [specifications](https://www.w3.org/2019/did-wg/PublStatus)
 
-A WG member is added to the group’s lists `public-did-wg@w3.org` and `member-did-wg@w3.org`; see the [separate section](#communications) for more details. Other mailing lists may be set up for task forces or other sub-committees; signing up to those list must be done manually.
+A WG member is added to the group’s lists `public-did-wg@w3.org` and
+`member-did-wg@w3.org`; see the [separate section](#communications) for
+more details. Other mailing lists may be set up for task forces or other
+sub-committees; signing up to those list must be done manually.
 
-Participation from the Public (i.e., non group members), via our Public e-mail lists is also welcome, provided comments, contributions, etc., are consistent with the [W3C Patent Policy](https://www.w3.org/TR/patent-policy/).
+Participation from the Public (i.e., non group members), via our Public
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
Participation from the Public (i.e., non group members) via our Public
```

File: WorkMode/index.md [hunk 10]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+The charter also states that:
 
-> …any resolution (including publication decisions) taken in a face-to-face meeting or teleconference will be considered provisional.
+> …any resolution (including publication decisions) taken in a face-to-face
+    meeting or teleconference will be considered provisional.
 >
-> A call for consensus (CfC) will be issued for all resolutions (for example, via email and/or web-based survey), with a response period from one week to 10 working days, depending on the chair's evaluation of the group consensus on the issue.
+> A call for consensus (CfC) will be issued for all resolutions (for
+    example, via email and/or web-based survey), with a response period
+    from one week to 10 working days, depending on the chair's evaluation
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
    from 5 to 10 working days, depending on the chair's evaluation
```

File: WorkMode/index.md [hunk 11]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+invitations for people who need them to obtain a visa, given sufficient
+notice.
+
+The consortium usually has an annual 
+[“Technical Plenary and All Working Group”](https://www.w3.org/2002/09/TPOverview.html)
+face-to-face meeting week (a.k.a “TPAC”) and this group typically has a f2f
+meeting during that week. The dates/locations are generally known a year or
+more in advance.
+
+For the minutes, resolutions, and consensus achieved at the f2f meetings
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
For the minutes, resolutions, and consensus achieved at the f2f meetings,
```

File: WorkMode/index.md [hunk 12]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
-
-It is also possible for people who are not members of the DID WG to attend face-to-face meetings as observers. Non-members have not made any commitment to provide standard W3C royalty-free licensing, so non-members are restricted to observer status only. Observers may listen, and participate in general discussions during the meeting. However, they must not make technical contributions, or attempt to influence an approach to a feature that may become part of the specification being discussed.
-
-If the public contributor, or the observer, works for a W3C member company, they are encouraged to ask their Advisory Committee (AC) representative to make them a DID WG participant.
-
-Please note that this is to provide as much protection as possible through the W3C Patent Policy. We take the royalty-free status of W3C standards very seriously, and any attempt to work-around these basic requirements would be considered a serious breach of meeting participation.
+It is possible for people who are not members of the DID WG to follow the
+Working Group's work by signing up to the group's public mailing list,
+read the mailing list archives, or watch the github issues. They can also
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
reading the mailing list archives, or watching the github issues. They can also
```

File: WorkMode/index.md [hunk 13]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+raise issues on github and the Working Group is required to answer to those
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
raise issues on github, which the Working Group is required to answer
```
(+ the following deletion of `issues`)

File: WorkMode/index.md [hunk 14]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+issues within a reasonable time. The chairs of the Working Group may also
```
Comment (TallTed) [inline WorkMode/index.md]: (along with the above line-edit, deleting `to those`)
```suggestion
within a reasonable time. The chairs of the Working Group may also
```

File: WorkMode/index.md [hunk 15]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+e.g., a Pull Request, see the
+[separate text](https://github.com/w3c/did-wg/blob/master/CONTRIBUTING.md)
+for the modalities.
+
+It is also possible for people who are not members of the DID WG to attend
+face-to-face meetings as observers. Non-members have not made any commitment
+to provide standard W3C royalty-free licensing, so non-members are restricted
+to observer status only. Observers may listen, and participate in general
+discussions during the meeting. However, they must not make technical
+contributions, or attempt to influence an approach to a feature that may
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
contributions, nor attempt to influence an approach to a feature, that may
```

File: WorkMode/index.md [hunk 16]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+become part of the specification being discussed.
+
+If the public contributor, or the observer, works for a W3C member company,
+they are encouraged to ask their Advisory Committee (AC) representative to
+make them a DID WG participant.
+
+Please note that this is to provide as much protection as possible through
+the W3C Patent Policy. We take the royalty-free status of W3C standards
+very seriously, and any attempt to work-around these basic requirements
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
very seriously, and any attempt to work around these basic requirements
```

File: WorkMode/index.md [hunk 17]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+
+Editors of the documents (as well as the chairs and the W3C staff) have the
+necessary access rights to make editorial changes on the specifications
+directly using the standard Git(Hub) commits and by merging pull requests.
+Other members of the group are encouraged to use the
+[“fork and pull model”](https://help.github.com/articles/about-collaborative-development-models/)
+when contributing to the text: work on a forked repository and issue a pull
+request on the main repository for that document when the contribution is
+ready. Editors should use the pull request mechanism (except for obvious,
+grammatical or stylistic changes), albeit they can choose to do that
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
grammatical, or stylistic changes), albeit they can choose to do that
```

File: WorkMode/index.md [hunk 18]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+directly on the core repository.
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
directly on the core repository (i.e., without creating a distinct fork).
```

File: WorkMode/index.md [hunk 19]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+
+(If you are new to GitHub, the
+[*“Introduction to Basic GitHub Contribution”*](https://iherman.github.io/misc-notes/docs/BasicGitHubContributionIntro)
+may be of help.)
+
+In line with the spirit of the asynchronous decision procedures outlined
+above, significant pull requests, as well as the closure of open issues,
+should be marked with a special label (to be defined) and left open for a
+week. If no objection is raised during that time the issue can be closed or
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
week. If no objection is raised during that time, the issue can be closed or
```

File: WorkMode/index.md [hunk 20]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+the pull request can be merged, respectively.
 
 #### Issue labels
-
-GitHub issues are also used as a records of wide reviews, of horizontal reviews, etc. The Working Group will define a number of labels (e.g., labeling an issue as part of the Horizontal Security review). Chairs, staff, and editors are responsible to set those labels accordingly. Similarly, when issues are waiting for external reviewers to react, labels will be used to signal the status of the issue. (See the [current set of labels](https://github.com/w3c/did-spec/labels).)
+GitHub issues are also used as a records of wide reviews, of horizontal
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
GitHub issues are also used as a record of wide reviews, of horizontal
```

File: WorkMode/index.md [hunk 21]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+status.
 
 ## Process
 
-The [Charter](https://www.w3.org{{ site.charter }}) and the [W3C Process Document](https://www.w3.org/Consortium/Process/) are the final arbiters of any process question, however the Working Group has adopted some complimentary, additional processes to aid in its productivity.
-
-* Adoption of [guiding technical principles](guiding_principles) towards quickly coming to consensus
+The [Charter](https://www.w3.org{{ site.charter }}) and the
+[W3C Process Document](https://www.w3.org/Consortium/Process/) are the
+final arbiters of any process question, however the Working Group has
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
final arbiters of any process question; however, the Working Group has
```

File: WorkMode/index.md [hunk 22]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+    work forward at a reasonable pace, we will not require most of the
+    group's decisions to be made via formal resolutions. 
+    * This will free the Editors to merge PRs and close Issues as soon as
+        they feel the group has reached consensus.
+    * Editors will attempt to reach out to interested parties, but it is up
+        to WG members to actively monitor progress.
+    * Any decision to merge may be challenged during a 48-hour window
+        following the decision.
+* It is encouraged that any Pull Request which changes the normative text
+    of the [DID Specification](https://github.com/w3c/did-spec) will be
```
Comment (TallTed) [inline WorkMode/index.md]:
```suggestion
    of the [DID Specification](https://github.com/w3c/did-spec) be
```

File: WorkMode/newbie.md [hunk 23]
```diff
@@ -4,43 +4,93 @@ layout: default
...
+specifications are developed, where issues are discussed. Even these pages
+are in a GitHub repository (look at the bottom for the reference). It is
+therefore important to familiarize yourself, at least at a basic level,
+with this tool. (If you are new to GitHub, the
+[*“Introduction to Basic GitHub Contribution”*](https://iherman.github.io/misc-notes/docs/BasicGitHubContributionIntro)
+may be of help.)
+
+Once you have your GitHub account name, **send this name to ivan@w3.org, or
+to one of the chairs**, who will add you to the Working Group’s “GitHub
+team” (in GitHub‘s jargon). That would give you the possibility of being
```
Comment (TallTed) [inline WorkMode/newbie.md]:
```suggestion
team” (in GitHub‘s jargon). That would enable you to be
```

File: WorkMode/newbie.md [hunk 24]
```diff
@@ -4,43 +4,93 @@ layout: default
...
+Recommendation track documents on GitHub, the system would know that you
+are a Working Group member, which means that the possible IPR issues are
+properly handled by virtue of your institution’s Working Group membership.
 
 ## IRC
 
-Meetings usually use [IRC](index#irc) as a tool for queue control and minute taking (and general chit-chat). **You should also send to ivan@w3.org your preferred IRC nickname (or nicknames, if you use several of those).** The goal is to improve the readability of the meeting minutes, so that third parties could also understand whose comments and remarks are in the minutes. Actually, in case you are already a seasoned GitHub user: the nicknames are stored (of course:-) in [file on GitHub](https://github.com/w3c/did-wg/blob/master/assets/nicknames.json), meaning that you can simply make a pull request providing with with your nickname(s) instead of a mail.
+Meetings usually use [IRC](index#irc) as a tool for queue control and
+minute taking (and general chit-chat). **You should also send ivan@w3.org
+your preferred IRC nickname (or nicknames, if you use several of those).**
```
Comment (TallTed) [inline WorkMode/newbie.md]:
```suggestion
your preferred IRC nickname(s).**
```

File: WorkMode/newbie.md [hunk 25]
```diff
@@ -4,43 +4,93 @@ layout: default
...
+The goal is to improve the readability of the meeting minutes, so that
+third parties can also understand whose comments and remarks are in the
+minutes. Actually, in case you are already a seasoned GitHub user: the
```
Comment (TallTed) [inline WorkMode/newbie.md]:
```suggestion
minutes. If you are already a seasoned GitHub user: the
```

File: WorkMode/newbie.md [hunk 26]
```diff
@@ -4,43 +4,93 @@ layout: default
...
+nicknames are stored (of course:-) in a
```
Comment (TallTed) [inline WorkMode/newbie.md]:
```suggestion
IRC and GitHub nicknames are stored (of course :-) ) in a
```

File: WorkMode/newbie.md [hunk 27]
```diff
@@ -4,43 +4,93 @@ layout: default
...
+
+New WG members may also be invited to introduce themselves during a 
+teleconference meeting.
 
 ## Questions, Comments, Issues, etc.?
 
-Please contact the group's Chairs and staff contacts (using the [group-did-wg-chairs@w3.org](mailto:group-did-wg-chairs@w3.org) list) — *at any time* — if you have any questions, issues, concerns, etc. about the Working Group.
+Please contact the group's Chairs and staff contacts (using the
+[group-did-wg-chairs@w3.org](mailto:group-did-wg-chairs@w3.org) list) — *at
+any time* — if you have any questions, issues, concerns, etc. about the
```
Comment (TallTed) [inline WorkMode/newbie.md]:
```suggestion
any time* — if you have any questions, issues, concerns, etc., about the
```

File: WorkMode/newbie.md [hunk 28]
```diff
@@ -4,43 +4,93 @@ layout: default
...
+Working Group.
 
 ## Resources
 
-If you are unfamiliar with processes and roles within the Consortium; there is an [introduction course](http://lists.w3.org/Archives/Public/www-archive/2014Apr/0026.html) available which takes little over an hour and a half to complete.
+If you are unfamiliar with processes and roles within the Consortium; there
+is an [introduction course](http://lists.w3.org/Archives/Public/www-archive/2014Apr/0026.html)
+available which takes little over an hour and a half to complete.
```
Comment (TallTed) [inline WorkMode/newbie.md]:
```suggestion
available which typically takes about 90 minutes to complete.
```

**Comment by TallTed** [comment]
I didn't realize there would be so many, or I would have made a more consolidated review.

**Comment by iherman** [comment]
Wow, @TallTed :-) Thanks!

File: WorkMode/index.md [hunk 29]
```diff
@@ -45,93 +74,230 @@ Participation from the Public (i.e., non group members), via our Public e-mail l
...
+e.g., a Pull Request, see the
+[separate text](https://github.com/w3c/did-wg/blob/master/CONTRIBUTING.md)
+for the modalities.
+
+It is also possible for people who are not members of the DID WG to attend
+face-to-face meetings as observers. Non-members have not made any commitment
+to provide standard W3C royalty-free licensing, so non-members are restricted
+to observer status only. Observers may listen, and participate in general
+discussions during the meeting. However, they must not make technical
+contributions, or attempt to influence an approach to a feature that may
```
Comment (brentzundel) [inline WorkMode/index.md]:
```suggestion
contributions, nor attempt to influence an approach, to a feature that may
```

**Comment by brentzundel** [comment]
> I didn't realize there would be so many, or I would have made a more consolidated review.

@TallTed fwiw, I really appreciated the use of `suggestion mode`
It makes your recommendations really easy to merge.

**Review by burnburn — APPROVED** [review]

**Pull request merged**